From 0e63c3b201281581c3976cb1fa16363577b018e9 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 4 Mar 2021 15:29:30 +0700
Subject: [PATCH 062/135] arm64: dts: renesas: r9a07g044l: add new property to
 set IO buffer direction in eth node

This commit adds a new property named "eth-pin-mode-switch" to
set IO buffer direction which is supported an api from pinctrl module.
Therefore, it is needed to parse the phandle to pinctrl node.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index 728a7e0..ace1710 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -293,6 +293,7 @@
 			phy-mode = "rgmii";
 			#address-cells = <1>;
 			#size-cells = <0>;
+			eth-pin-mode-switch = <&pinctrl 0>;
 			status = "disabled";
 		};
 		
@@ -308,6 +309,7 @@
 			phy-mode = "rgmii";
 			#address-cells = <1>;
 			#size-cells = <0>;
+			eth-pin-mode-switch = <&pinctrl 1>;
 			status = "disabled";
 		};
 	};
-- 
2.7.4

